### PR TITLE
Create weekly-update.2015-07-03.md

### PR DESCRIPTION
**Pull request by diagramatics**
I took the liberty of writing a format that the Markdown result can be pasted to Medium and it'll retain the right formatting on it, including the subtitle. I'd be happy to change it back if it doesn't suit tastes.

### CODE REVIEW

**Comment by Gioyik** [comment]
Our weekly updates are not only about io.js anymore. We use from conciliation: "io.js and Node.js News" as title.

**Comment by diagramatics** [comment]
The title never gets used on the Medium posts though, which is why I did the original one.

So how about "io.js and Node.js Week of July 3rd" or "io.js and Node.js News — July 3rd"?

File: weekly-updates/weekly-update.2015-07-03.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* **v8**: Fixed an out-of-band write in utf8 decoder. **This is an important security update** as it can be used to cause a denial of service attack.
+* **openssl**: Upgrade to 1.0.2b and 1.0.2c, introduces DHE man-in-the-middle protection (Logjam) and fixes malformed ECParameters causing infinite loop (CVE-2015-1788). See the [security advisory](https://www.openssl.org/news/secadv_20150611.txt) for full details. (Shigeki Ohtsu) [#1950](https://github.com/nodejs/io.js/pull/1950) [#1958](https://github.com/nodejs/io.js/pull/1958)
+* **build**:
+  - Added support for compiling with Microsoft Visual C++ 2015
+  - Started building and distributing headers-only tarballs along with binaries
+
+#### 2.3.2
+
+* **build**:
+  * Added support for compiling with Microsoft Visual C++ 2015
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-07-03.md:23]: is this `*` ? 

in v1.8.3 section : 

``` markdown
* **build**:
  - Added support for compiling with Microsoft Visual C++ 2015
  - Started building and distributing headers-only tarballs along with binaries
```

it seems to be better `-` .

**Comment by yosuke-furukawa** [comment]
Thank you @diagramatics 
LGTM when you fix the nits.

File: weekly-updates/weekly-update.2015-07-03.md (hunk 1 above)
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-07-03.md:23]: Uh, does this mean you'd like it to be using `-` or `*` for the nested list items?

Oh, and worth pointing out, nested lists are not available in Medium too. It'll just be reformatted to the parent's list item.

**Comment by bnb** [comment]
LGTM. :shipit:

File: weekly-updates/weekly-update.2015-07-03.md (hunk 1 above)
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-07-03.md:23]: I mean I would like to use same symbol to list items.
In v1.8.3 section, you use `-`. but In v2.8.3 section, you use `*`.

v1.8.3:

```
* **build**:
  - Added support for compiling with Microsoft Visual C++ 2015
  - Started building and distributing headers-only tarballs along with binaries
```

v 2.3.2:

```
* **build**:
  * Added support for compiling with Microsoft Visual C++ 2015
  * Started building and distributing headers-only tarballs along with binaries
```

I don't have a strong opinion which symbol (`*`, `-`) is better.

**Comment by diagramatics** [comment]
Changed the title for a better one and edited the formatting. Hopefully the title sticks.

**Comment by yosuke-furukawa** [comment]
Thank you!!

File: weekly-updates/weekly-update.2015-07-03.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+### io.js and Node.js News — July 3rd
+Important patches for io.js 1.8 and 2.3 and upcoming events.
+
+### io.js 1.8 and 2.3 Releases
+
+This week we have three io.js releases: [v2.3.2](https://iojs.org/dist/v2.3.3/) and two following important security patches [v1.8.3](https://iojs.org/dist/v1.8.3/) and [v2.3.3](https://iojs.org/dist/v2.3.3/), complete changelog from previous releases can be found [on GitHub](https://github.com/nodejs/io.js/blob/master/CHANGELOG.md) with the [v1.x changelog here](https://github.com/nodejs/io.js/blob/v1.x/CHANGELOG.md).
```
Comment (c0b41) [inline weekly-updates/weekly-update.2015-07-03.md:6]: v2.3.3 must v2.3.2

Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-07-03.md:6]: oops. thanks.

Comment (diagramatics) [inline weekly-updates/weekly-update.2015-07-03.md:6]: Ah, darn it. Apologies.

**Pull request merged**